docs/usage/dashboards.md

```diff
@@ -122,9 +122,22 @@ The widget configuration will be saved as a JSON file, with the title formatted
 Dashboards can be imported from the custom dashboards list:
 
 1. Click on the `Import dashboard` button in the right top corner
+   ![Import dashboard option](assets/import-dashboard-new.png)
 2. Select your file.
 
-![Import dashboard option](assets/import_dashboard.png)
+You can select dashboards file from the XTM Hub by clicking the ```Import from Hub``` button
+
+_Connect to the XTM Hub to download a JSON file that will be easy to import on OpenCTI following this process._
+
+#### 1Click deploy a dashboard
+From the XTM Hub, you can effortlessly deploy your desired dashboard with just a single click.
+To get started, simply enroll your OpenCTI platform following the instructions in our Enrollment documentation here: XXX.
```

Comment:
We need to add the link to our documentation before merging this PR then!

Here, in order to be more clear, we should specify that the enrollment is inside of the documentation of an other product: xtm hub